services/squareone/values.yaml

```diff
@@ -0,0 +1,105 @@
+# Default values for squareone.
+#
+# Global parameters will be set by parameters injected by Argo CD and should
+# not be set in the individual environment values files.
+global:
```

Comment:
I generally move these to the bottom of values.yaml because people never need to set them or see them.